enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/HighAvailabilityMemberStateMachine.java

```diff
@@ -237,7 +239,12 @@ public void memberIsUnavailable( String role, InstanceId unavailableId )
                  HighAvailabilityModeSwitcher.SLAVE.equals( role ) &&
                  state == HighAvailabilityMemberState.SLAVE )
             {
-                changeStateToPending();
+                String loggingMessage = "Got memberIsUnavailable(" + unavailableId + ")";
```

Comment:
imho it is better to log things right here and not pass partial messages to `changeStateToPending` method